src/txHashBuilder_test.c

```diff
@@ -123,13 +125,14 @@
+A700848258200B40265111D8BB3C3C608D95B3A0BF83461ACE32D79336579A1939B3AAD1C0B7008258201B40265111D8BB3C3C608D95B3A0BF83461ACE32D79336579A1939B3AAD1C0B7018258202B40265111D8BB3C3C608D95B3A0BF83461ACE32D79336579A1939B3AAD1C0B7028258203B40265111D8BB3C3C608D95B3A0BF83461ACE32D79336579A1939B3AAD1C0B703018582582B82D818582183581C6EE5BB111C8771CE03278E624056A12C9CFB353EB112E8ABF21FA4FEA0001A74EEE4081864825839009493315CD92EB5D8C4304E67B7E16AE36D61D34502694657811A2C8E32C728D3861E164CAB28CB8F006448139C8F1740FFB8E7AA9E5232DC18C8825823409493315CD92EB5D8C4304E67B7E16AE36D61D34502694657811A2C8E87688F50973819012C82581D609493315CD92EB5D8C4304E67B7E16AE36D61D34502694657811A2C8E19019082581D609493315CD92EB5D8C4304E67B7E16AE36D61D34502694657811A2C8E1901F402182A031A000395F8048782008200581C32C728D3861E164CAB28CB8F006448139C8F1740FFB8E7AA9E5232DC82018200581C32C728D3861E164CAB28CB8F006448139C8F1740FFB8E7AA9E5232DC82018200581C337B62CFFF6403A06A3ACBC34F8C46003C69FE79A3628CEFA9C472518A03581C5631EDE662CFB10FD5FD69B4667101DD289568E12BCF5F64D1C406FC5820198890AD6C92E80FBDAB554DDA02DA9FB49D001BBD96181F3E07F7A6AB0D06401A1DCD65001A1443FD00D81E820101581DE03A7F09D3DF4CF66A7399C2B05BFA234D5A29560C311FC5DB4C49071181581C3A7F09D3DF4CF66A7399C2B05BFA234D5A29560C311FC5DB4C4907118384001904D24408080808F68301F678204141414120323430303A636230303A323034393A313A3A613239663A31383034820278204141414120323430303A636230303A323034393A313A3A613239663A3138303482781968747470733A2F2F746573747374616B65706F6F6C2E636F6D5820914C57C1F12BBF4A82B12D977D4F274674856A11ED4B9B95BD70F5D41C5064A683028200581C32C728D3861E164CAB28CB8F006448139C8F1740FFB8E7AA9E5232DC581C0D13015CDBCDBD0889CE276192A1601F2D4D20B8392D4EF4F9A754E283028200581C32C728D3861E164CAB28CB8F006448139C8F1740FFB8E7AA9E5232DC581C1D13015CDBCDBD0889CE276192A1601F2D4D20B8392D4EF4F9A754E283028200581C32C728D3861E164CAB28CB8F006448139C8F1740FFB8E7AA9E5232DC581C2D13015CDBCDBD0889CE276192A1601F2D4D20B8392D4EF4F9A754E205A1581DE032C728D3861E164CAB28CB8F006448139C8F1740FFB8E7AA9E5232DC19029A075820DEADBEEFDEADBEEFDEADBEEFDEADBEEFDEADBEEFDEADBEEFDEADBEEFDEADBEEF
```

Comment:
for the sake of consistency, this should be lowercase

Reply:
I'd rather keep it as it is, since uppercase is what comes out of the ledger via the usb tool (perhaps that could be changed, but likely not worth the trouble) and what is used for comparison. We already have a mix of upper- and lowercase in the file, depending on where the data come from.

Reply:
I've found that the upper/lowercase was not consistent (mostly lowercase), so I ended up changing this.